src/manage_nsfs/nc_lifecycle.js

```diff
@@ -149,7 +149,8 @@ async function process_buckets(config_fs, bucket_names, system_json) {
  */
 async function process_bucket(config_fs, bucket_name, system_json) {
     const bucket_json = await config_fs.get_bucket_by_name(bucket_name, config_fs_options);
-    const account = { email: '', nsfs_account_config: config_fs.fs_context, access_keys: [] };
+    const account_path = config_fs.get_account_path_by_id(bucket_json.owner_account);
```

Comment:
1. There is already a function that does it for you - get_identity_by_id(), notice that this function is backwards compatible, therefore it's always the function you should choose
2. Please handle errors, for example the account owner was deleted - we don't want all the lifecycle run to fail

Reply:
the case where account is deleted might be tricky. we currently don't handle well any uid/gid which doesn't belong to the bucket owner. we wouldn't be able to add delete markers properly